lighthouse-core/report/html/templates.html

```diff
@@ -164,11 +164,13 @@
       width: 100%;
       will-change: transform;
     }
+    .lh-narrow .lh-header-bg {
+      height: var(--report-header-height-narrow);
+    }
     .lh-lighthouse {
       position: absolute;
-      top: var(--report-header-height);
-      right: 50%;
-      transform: translate3d(var(--report-width-half), -100%, 0);
+      bottom: -4px;
```

Comment:
bottom -4?

Reply:
The lighthouse SVG element is 4px above the bottom of the svg's wrapper div (which in this screenshot is set to 0px bottom) so I moved it 4px down to match the header height exactly.  I tried to fix the SVG but hand editing an SVG proved a bit difficult.  So I moved the whole thing 4 px down.
![image](https://user-images.githubusercontent.com/6392995/50355880-e61a4480-0504-11e9-8683-9e350a92b703.png)